mozci/console/commands/push.py

```diff
@@ -199,33 +199,32 @@ def handle(self) -> None:
                 self.line("-" * 50)
 
             if output:
+
+                def _serialize_regressions(regressions):
+                    return {
+                        group: [
+                            {
+                                "task_id": task.id,
+                                "label": task.label,
+                                "autoclassify": task.autoclassify,
+                            }
+                            for task in failing_tasks
+                        ]
+                        for group, failing_tasks in regressions.items()
+                    }
+
                 to_save = {
+                    "autoclassification": config["autoclassification"]["enabled"],
```

Comment:
For simplicity sake we can avoid this, as `autoclassify` for individual groups will be false anyway when autoclassification is disabled.